astropy/wcs/wcsapi/tests/test_fitswcs.py

```diff
@@ -696,13 +714,13 @@ def test_time_1d_location_incomplete(header_time_1d_noobs):
     assert time.location is None
 
 
-def test_time_1d_location_unsupported(header_time_1d):
+def test_time_1d_location_unsupported(header_time_1d_noobs):
```

Comment:
Maybe better with extra underscore, as "noobs" can be misinterpreted:
```suggestion
def test_time_1d_location_unsupported(header_time_1d_no_obs):
```

Reply:
Done! (had to change 16 other entries so did it directly instead of using the suggestion)

Reply:
That's fine; wasn't expecting you to anyway. Thanks!